Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -213,8 +220,13 @@ bool EventRegionContext::shouldConsolidateInteractionRegion(RenderObject& render
 
         auto& ancestorBounds = discoveredIterator->value.first().rectInLayerCoordinates;
 
+        constexpr int loosenContainmentCheck = 3;
```

Comment:
`looseContainmentMargin` maybe?
and make it a float since `rectInLayerCoordinate` is a FloatRect